editor/src/components/canvas/canvas-strategies/strategies/grid-helpers.ts

```diff
@@ -779,17 +778,19 @@ export function getGridRelativeContainingBlock(
 
   // Gap needs to be set only if the other two are not present or we'll have rendering issues
   // due to how measurements are calculated.
-  if (
-    gridMetadata.specialSizeMeasurements.rowGap != null &&
-    gridMetadata.specialSizeMeasurements.columnGap != null
-  ) {
-    const gap = gridMetadata.specialSizeMeasurements.gap
-    gridElement.style.gap = gap == null ? 'initial' : `${gap}px`
-  } else {
+  const hasRowGap = gridMetadata.specialSizeMeasurements.rowGap != null
```

Comment:
this is a fix tangential to this PR, the problem is that `gap` itself might be null even if `rowGap` and `columnGap` are defined - the example is a configuration with uneven row/col gaps.